clients/drcachesim/common/options.cpp

```diff
@@ -982,6 +982,13 @@ droption_t<bool> op_encodings2regdeps(
     "This option is for -simulator_type " RECORD_FILTER ". When present, it converts "
     "the encoding of instructions from a real ISA to the DR_ISA_REGDEPS synthetic ISA.");
 
+droption_t<std::string>
+    op_filter_func_markers_by_value(DROPTION_SCOPE_FRONTEND,
+                                    "filter_func_markers_by_value", "",
+                                    "Comma-separated integers of function IDs to keep.",
```

Comment:
Please add what you have above: "This option is for -simulator_type " RECORD_FILTER "".
Actually `s/simulator_type/tool/`; replace above too.

Reply:
Done.